app/views/email_authentications/shared/_links.html.erb

```diff
@@ -1,21 +1,21 @@
 <%- if controller_name != 'sessions' %>
-  <%= link_to "Login", new_session_path(resource_name) %><br />
+  <i class="fas fa-lock mr-1 text-muted"></i> <%= link_to "Login", new_session_path(resource_name) %><br />
 <% end -%>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
   <%= link_to "Sign up", new_registration_path(resource_name) %><br />
 <% end -%>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
+  <i class="far fa-question-circle mr-1 text-muted"></i> <%= link_to "Forgot your password?", new_password_path(resource_name) %><br />
 <% end -%>
 
 <%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
   <%= link_to "Didn't receive confirmation instructions?", new_confirmation_path(resource_name) %><br />
 <% end -%>
 
 <%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
-  <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
+  <i class="fas fa-envelope mr-1 text-muted"></i> <%= link_to "Didn't receive password reset instructions?", new_unlock_path(resource_name) %><br />
```

Comment:
This link doesn't trigger a password reset, it re-sends an "unlock" email. User accounts can get locked if they have several failed login attempts in quick succession. Let me send you some screenshots of what that flow looks like to the user, and we can decide what to do with this link.

Reply:
Can you suggest pithy wording that will make sense for normal humans?

Reply:
Something that might be common on other sites/apps?

Reply:
![Screen Shot 2021-01-08 at 4 13 31 PM](https://user-images.githubusercontent.com/4241399/104011104-997d3100-51d3-11eb-93aa-67d07aeeb902.png)

This is what users see today.

We could do a couple things:
1. Ideally, stick the link in the flash message:
> Your account is locked. Check your email for instructions on how to unlock your account. If you didn't receive instructions, [click here](https://example.com).

2. Enhance the flash message a little bit, and keep the link where it is. Only show the link when the user is on this "account locked" screen.
Flash:
> Your account is locked. Check your email for instructions on how to unlock your account.

Link:
> [Didn't receive instructions to unlock your account?](https://example.com)

I'm suggesting (2) because (1) is really awkward code-wise. Putting links into the flash messages is a bit gnarly.

Reply:
2 makes tons of sense! Good call @harimohanraj89